.github/workflows/python_unit_test.yaml

```diff
@@ -7,14 +7,33 @@ on:
         description: 'A URL identifying a Bash script that can run commands needed for service-specific test setup'
```

Comment:
maybe `setup-script-location` should be optional?

Reply:
Not all repositories need special setup to run the tests, for example here running the pytest command is enough: https://github.com/uc-cdis/gen3utils/blob/ac5d931/.travis.yml

I would even make it default to a path like `tests/ci_setup.sh` so that it's consistent in all the repos and we only overwrite it if necessary, and just skip the step if this file doesn't exist